Backend/config.json

```diff
@@ -0,0 +1,112 @@
+{
```

Comment:
Please drop all profiles that currently useless. Keep only local and dev. Thank you in advance!